### PR TITLE
Connect to gRPC server transcoding

### PR DESCRIPTION
**Pull request by emcfarlane**
Adds `connect.GRPCHandler` as a crosstesting option.

Currently WIP testing for https://github.com/bufbuild/connect-go/pull/527

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -130,6 +131,33 @@ services:
     entrypoint: /usr/local/bin/client --host="server-grpc" --port="8083" --implementation="connect-grpc-h2" --cert "cert/client.crt" --key "cert/client.key"
     depends_on:
       - server-grpc
+  client-connect-to-server-grpc-adapter-h1:
```
Comment (smaye81) [inline docker-compose.yaml:134]: @emcfarlane just a dumb nit, but in my #477 PR, I tried to make the names a bit more scalable and flexible once additional languages and implementations are added. I think we settled on:

`client-{client implementation / protocol}-to-server-{server implementation}-{http version}`

where client implementation / protocol would be one of:

```
grpc-go (gRPC Go client)
grpc-web (gRPC-Web client)
connect-go (Connect-Go client using Connect protocol)
connect-go-grpc (Connect-Go client using gRPC protocol)
connect-go-grpc-web (Connect-Go client using gRPC-Web protocol)
connect-web (Connect-Web client using Connect protocol)
connect-web-grpc-web (Connect-Web client using gRPC-Web protocol)
...etc
```

Can you rename these containers to be:
```
client-connect-go-to-server-grpc-adapter-h1
client-connect-go-to-server-grpc-adapter-h2
client-connect-go-grpc-web-to-server-grpc-adapter-h2:
```

That should keep things consistent once #477 lands.  Thanks!

Comment (emcfarlane) [reply docker-compose.yaml:134]: Yep will do!

**Comment by nicksnyder** [comment]
Just curious, what is the state of this? (my question is not an implicit request to prioritize this)

**Comment by emcfarlane** [comment]
@nicksnyder this was for connect-go grpc-adapter package. We are moving that project so closing this for now!

**Pull request closed without merging**